src/plugins/data/public/plugin.ts

```diff
@@ -35,6 +36,7 @@ export class DataPublicPlugin implements Plugin<DataPublicPluginSetup, DataPubli
   public start(core: CoreStart): DataPublicPluginStart {
     return {
       autocomplete: this.autocomplete,
+      getSuggestions: getSuggestionsProvider(core.uiSettings, core.http),
```

Comment:
In `data` plugin we have this services pattern where each service is returned from life-cycles, here `autocomplete` and `getSuggestions` are some utilities. Should we collect them into a service, say `suggestions`?

Reply:
I actually thought of placing it inside the `automplete` service. 
I just wasn't sure and didn't want to create unnecessary code while not sure.
Until we have a good hunch, I think the services pattern is great for more complex things, than a single function.